pkg/extensions/customresources.go

```diff
@@ -20,25 +20,25 @@ import (
 	"fmt"
 	"time"
 
+	"github.com/sirupsen/logrus"
+	autoscalingv1 "k8s.io/api/autoscaling/v1"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/api/meta"
+	"k8s.io/apimachinery/pkg/runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
```

Comment:
Did the project switch the general guideline for import orders? It was more or less "std, project, external" so far and this turns it into "std, external, project".

Reply:
Well, the guidelines we agreed on was to accept the import order as long as `goimports` doesn't complain.
Sometimes my IDE put stuff in reverse order, sometimes it does not. I don't know why, but I also don't care to much, as both is accepted by `goimports`...

Reply:
Okay then @rfranzke should not have told me to re-order imports at all :-D I was under the impression that there is a somewhat agreed upon order in gardener/* repos.